indigo/analysis/work_detail/base.py

```diff
@@ -45,3 +45,18 @@ def work_friendly_type(self, work):
             return _(uri.subtype.upper())
 
         return _('Act')
+
+
+class ChapterWorkDetail(BaseWorkDetail):
+    """ Takes Chapter numbers into account when working out numbered title.
+    """
+    def work_numbered_title(self, work):
+        work_type = super().work_friendly_type(work)
+        if work.properties.get('cap'):
+            # eg. Chapter 2
+            return f"Chapter {work.properties['cap']}"
```

Comment:
This should be localisable: `return _('Chapter %(cap)s') % {'cap': work.properties['cap']}`